packages/smooth_app/lib/pages/product/add_new_product_page.dart

```diff
@@ -159,24 +177,30 @@ class _AddNewProductPageState extends State<AddNewProductPage> {
               _isProductLoaded = true;
             });
           }
-          initialPhoto.delete();
         },
       ),
     );
   }
 
   Widget _buildImageUploadedRow(
```

Comment:
Could your create a dedicated Widget please?

Reply:
Hello @g123k 
This block of code is not here anymore, I can't understand whether you want me to write it again here or what,? 

Also, I looked into issue #2843. So, since we delete the cropped image from the cache when it's done uploading, the flutter engine doesn't find any file at that location to render that image , so if we don't remove that image explicitly, is it going to be removed on its own ???

Reply:
Ok then forget about this

Reply:
Okay, but my question was 

### **if we don't remove that image explicitly, is it going to be removed on its own ???**

Reply:
No you have to delete it manually